spec/stat_tracker_spec.rb

```diff
@@ -323,10 +337,42 @@
             expect(@stat_tracker.worst_coach).to eq('ur mom')
         end
     end
+
```

Comment:
This is a great use of mocking data to avoid using the entire csv file, saving us time and efficiency.